app/assets/javascripts/app/views/agent_ticket_create.jst.eco

```diff
@@ -30,7 +30,7 @@
             <input type="hidden" name="form_id" value="<%= @form_id %>"/>
 
             <div class="ticket-form-top"></div>
-            <div class="article-form-top"></div>
+            <div id="editor" class="article-form-top"></div>
```

Comment:
We can't work with id here. Because the ticket create screen can be opened many times as you want.